boilerplates/aws/files/cdk/bin/infrastructure.ts

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env node
+import "source-map-support/register";
+import * as cdk from "aws-cdk-lib";
+import { VikeStack } from "../lib/vike-stack";
+
+import * as certificatemanager from "aws-cdk-lib/aws-certificatemanager";
+import * as route53 from "aws-cdk-lib/aws-route53";
+
+// eslint-ignore
```

Comment:
@magne4000 Any idea on how to handle typescript type check and listing on an import `import { stackNameSuffix } from "../stack-name-suffix.json";` which is a file `$stack-name-suffix.json.ts` which gets created on template creation?

Reply:
There is a workaround to remove the comments in the generated file at least.
Just precede the comments with `//# BATI.has("REMOVE-COMMENT") || "remove-comments-only"`:

```
//# BATI.has("REMOVE-COMMENT") || "remove-comments-only"`
// eslint-ignore
// @ts-ignore file will be generated by the transformer
import { stackNameSuffix } from "../stack-name-suffix.json";
```

Reply:
(so no actual way to handler typescript properly in IDE there)